controllers/user.js

```diff
@@ -32,14 +39,38 @@ module.exports = class User {
       res.status(500).json({ error: error.message });
     }
   }
-
-  static async apiUpdateOneuser(req, res, next) {
+  //Update
+  static async apiUpdateUserPassword(req, res, next) {
     try {
+      const { id, password } = req.body;
+      if (!id) {
+        res.status(401);
+      }
+      const user = await UserService.updateUser({ id, password });
+      if (user == null) {
+        res.status(501).json( {error: "unknown error"});
+      }
+      res.status(206).json(user);
     } catch (error) {
-      res.status(500).json({ error: error });
+      res.status(500).json({ error: error.message });
     }
   }
-
+  static async apiUpdateUserPhone(req, res, next) {
+    try {
+      const { id, phone } = req.body;
+      if (!id) {
+        res.status(401);
+      }
+      const user = await UserService.updateUser({ id, phone });
+      if (user == null) {
+        res.status(501).json( {error: "unknown error"});
+      }
+      res.status(206).json(user);
+    } catch (error) {
+      res.status(500).json({ error: error.message });
+    }
+  }
+  //Delete
```

Comment:
`apiUpdateUserPhone` and `apiUpdateUserPassword` is both using updateUser which is kind of duplicated, and updating the phone and password can both use the same `UserService.updateUser` to update. 

For now this is fine.